From 6bb056aa8359e33f9b6f8974e2cc8b518ca9fe96 Mon Sep 17 00:00:00 2001
From: Li xin <lixin.fnst@cn.fujitsu.com>
Date: Tue, 18 Aug 2015 12:34:32 +0900
Subject: [PATCH] pacemaker: search header from STAGING_INCDIR to walkaround
 error

 ** -I/usr/include/ is unsafe for cross-compilation

 ... ...

 ** This autoconf log indicates errors, it looked at host include and/or library paths
 ** while determining system capabilities

Upstream-Status: pending

Signed-off-by: Bian Naimeng <biannm@cn.fujitsu.com>
Signed-off-by: Li Xin <lixin.fnst@cn.fujitsu.com>
---
 configure.ac | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 08b4611..6fcd1aa 100644
--- a/configure.ac
+++ b/configure.ac
@@ -463,7 +463,8 @@ esac
 
 dnl Eventually remove this
 if test "$cross_compiling" != "yes"; then
-   CFLAGS="$CFLAGS -I${prefix}/include/heartbeat"
+   CFLAGS="$CFLAGS -I${STAGING_INCDIR}/heartbeat"
+   CPPFLAGS="$CPPFLAGS -I${STAGING_INCDIR}/heartbeat"
 fi
 
 AC_SUBST(INIT_EXT)
-- 
1.8.4.2

